source/grid/grid_in.cc

```diff
@@ -867,6 +867,18 @@ void GridIn<dim, spacedim>::read_abaqus (std::istream
     {
       read_ucd(in_ucd, apply_all_indicators_to_manifolds);
     }
+  catch (std::exception &exc)
+    {
+      std::cerr
+          << "Exception on processing internal UCD data: " << std::endl
+          << exc.what()
+          << std::endl;
+
+      AssertThrow(false, ExcMessage("Internal conversion from ABAQUS file to UCD format was unsuccessful. \
+                                   More information is provided in an error message printed above. \
```

Comment:
I'm coming too late, but multiline string constants are not standard C++, and the amount of spacing you will get here is awkward as well. Can you just use quotes for each line separately, as we do elsewhere?